ocs/ocs_agent.py

```diff
@@ -694,8 +694,10 @@ def start(self, op_name, params=None):
                     handler = ParamHandler(params)
                     params = handler.batch(op.launcher._ocs_prescreen)
                 except ParamError as err:
+                    self.log.error(f"Caught ParamError during start call: {err}")
                     return (ocs.ERROR, err.msg, {})
                 except Exception as err:
+                    self.log.error(f"Caught Exception during start call: {err}")
```

Comment:
Passing a string containing random repr(exception) to log.error seems to almost always lead to a crash, because repr(err) has some {} in it, and log.error tries to string interpolate it.  I believe
```
self.log.error("Caught Exception during start call: {err}", err=err)
```
is how you're supposed to do it.

Reply:
Mhm, I remember seeing that crash often a while back, but not so much these days, I wonder if something changed. I did test the f-string evaluation (see below) and it was working, but I've change to use the standard txaio syntax anyway.

```
>>> import txaio
>>> txaio.use_twisted()
>>> txaio.start_logging()
>>> log = txaio.make_logger()
>>> log.info('test')
2023-05-24T10:27:40-0400 test
>>> log.info('test {foo}', foo='bar')
2023-05-24T10:29:51-0400 test bar
>>> foo='bar'
>>> log.info(f'test {foo}')
2023-05-24T10:28:20-0400 test bar
```

Reply:
Ahh, I see, that's a helpful example. Thanks!